telegram/passport/encryptedpassportelement.py

```diff
@@ -19,8 +19,8 @@
 """This module contains an object that represents a Telegram EncryptedPassportElement."""
 from base64 import b64decode
 
-from telegram import (TelegramObject, PassportFile, PersonalDetails, IdDocumentData,
-                      ResidentialAddress)
+from telegram import (IdDocumentData, PassportFile, PersonalDetails,
+                      ResidentialAddress, TelegramObject)
```

Comment:
Any particular reasons for this change?

Reply:
Probably just my IDE sorting imports alphabetically

Reply:
Okay. I recall Noam saying something about keeping the diff small, then again with this PR … I'll resolve.